receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,533 @@
+    unit: "{virtual_machines}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [vm_count_power_state]
+  vcenter.cluster.host.count:
+    enabled: true
+    description: The number of hosts in the cluster
+    unit: "{hosts}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [host_effective]
+
+  vcenter.datastore.disk.usage:
+    enabled: true
+    description: The amount of space in the datastore.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [disk_state]
+  vcenter.datastore.disk.utilization:
+    enabled: true
+    description: The utilization of the datastore
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the host system
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.cpu.usage:
+    enabled: true
+    description: The amount of CPU in Hz used by the host
+    unit: "MHz"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.disk.throughput:
+    enabled: true
+    description: The throughput to the host system's disk
+    unit: By/s
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.disk.latency.avg:
+    enabled: true
+    description: The latency of operations to the host system's disk.
+    unit: ms
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [latency_direction, latency_type]
+    extended_documentation: This latency is the sum of the device and kernel write latencies. Requires Performance Counter level 2 for metric to populate.
+  vcenter.host.disk.latency.total:
+    enabled: true
+    description: The total reported total latency (device and kernel times)
+    unit: ms
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [latency_direction]
+  vcenter.host.memory.utilization:
+    enabled: true
+    description: The percentage of the host system's memory capacity that is being utilized
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.memory.usage:
+    enabled: true
+    description: The amount of memory the host system is using
+    unit: MBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.network.throughput:
+    enabled: true
+    description: The amount of data that was sent or received over the network by the host
+    unit: "{KBy/s}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [throughput_direction]
+  vcenter.host.network.usage:
```

Comment:
Is the just the sum of the `vcenter.host.network.throughput` data points?

Reply:
So this metric is tied to the internal perf metric: 

vcenter.host.network.usage => net.usage.average
> Network utilization (combined transmit-rates and receive-rates) during the interval

vcenter.host.network.throughput => net.bytes{Rx|Tx}.average
> Average amount of data received per second
> Average amount of data transmitted per second

If can see an argument of removing the `vcenter.host.network.usage` if you think that we get the same information from aggregation of the `vcenter.host.network.throughput` metric. Any thoughts on keeping/removing? 

doc reference: https://www.virten.net/2020/12/vsphere-7-0-performance-counter-description/

Reply:
They do seem to be representing roughly the same information in different ways. 

If `vcenter.host.network.throughput` is a monotonic counter, then I think it is sufficient. Average values over undefined timeframes are of limited usefulness.